### ISSUE TITLE
com.gargoylesoftware.htmlunit.ObjectInstantiationException: unable to create HTML parser (NativeException)

### ISSUE THREAD

**Issue by w1150n** [jarib/celerity#56]
I've just installed the follow gem versions on JRuby 1.6.4:

capybara (1.0.1)
mime-types (>= 1.16)
nokogiri (>= 1.3.3)
rack (>= 1.0.0)
rack-test (>= 0.5.4)
selenium-webdriver (~> 2.0)
xpath (~> 0.1.4)
capybara-celerity (0.0.1)
capybara (>= 0.4.1)
celerity (>= 0.7.9)
celerity (0.9.0)

and get the error: "com.gargoylesoftware.htmlunit.ObjectInstantiationException: unable to create HTML parser (NativeException)"

which seems to cause the following error for each case:

"java.lang.NullPointerException: No document element ("

Any ideas?

Cheers,
Matt

**Comment by w1150n** [comment]
Hm, this seems to be a gem require problem caused by requiring capybara before celerity.  The error above went away for me after changing my requires to:

require 'celerity'
require 'capybara'
require 'capybara/celerity'

**Comment by w1150n** [comment]
Reopening this since it's still a bug.

**Comment by lukas2** [comment]
I can confirm this issue. In my case I required the "mechanize" gem before celerity. If I require celerity first, it works.

**Comment by hexagile** [comment]
Yes, I can also confirm with JRuby 1.6.3 and both celerity 0.9.0 and 0.9.1. However, it's OK with Celerity 0.8.9 so I reverted to that.
